### ISSUE TITLE
port extension latex_envs to jupyterhub

### ISSUE THREAD

**Issue by epifanio** [ipython-contrib/IPython-notebook-extensions#619]
Hi,

I'm migrating my environment to a multiuser jupyterhub, I'm having troubles in loading the ```latex_envs``` extension, I guess because the code is using the path to ```htpp://my.server.host:port/hub/``` instead of ```htpp://my.server.host:port/user/username```.

in the *codefolding* extension, i saw some [smart use of the config utility](https://github.com/ipython-contrib/IPython-notebook-extensions/blob/master/nbextensions/usability/codefolding/main.js#L12-L25) which perhaps can be re-used in the latex_envs code, the error in the js console give a 404 for ```thmslnNb4.js``` which I guess is called in:  https://github.com/ipython-contrib/IPython-notebook-extensions/blob/master/nbextensions/usability/latex_envs/latex_envs.js#L48-L53

**Comment by jcb91** [comment]
> i saw some smart use of the config utility 

Your link here doesn't seem to point to lines using the config utility, so I'm not sure what you mean.

Yep, I think this bug is basically down to the hardcoded `nbextensions/...` urls in [nbextensions/usability/latex_envs/latex_envs.js#L50-L52](https://github.com/ipython-contrib/IPython-notebook-extensions/blob/93d19ac20a1af4f56a09bd5fe064346e616edba8/nbextensions/usability/latex_envs/latex_envs.js#L50-L52) - if you replace these with relative urls (so in this case, that should give `"./thmsInNb4"`, `"./bibInNb4"`, and `"./initNb"` respectively), I think requirejs should figure out the rest correctly?

**Comment by epifanio** [comment]
sorry I meant utils:

```var base_url = utils.get_body_data("baseUrl");``` 

which can get the ```base_url```.

I'll try now your suggestions, thanks.

**Comment by jcb91** [comment]
> sorry I meant utils:

Right, I see what you mean. However, I think relative paths are still the way forward, in order not to hard-code a required install location into the extension.

**Comment by epifanio** [comment]
Yes I confirm your right, using:

```
define(["require", 
	'base/js/namespace',
	"./thmsInNb4.js",
	"./bibInNb4.js",
	"./initNb.js"], function (require,Jupyter, thmsInNb, bibsInNb,initNb) {    
```

it now loads the extensions correctly (only for the main user, the admin of the Hub) #617

**Comment by jcb91** [comment]
I guess this should now be fixed by #621, so I'll close. Of course reopen if it's not fixed after all!
